### PR TITLE
feat: Lokalize Validation Messages

### LINKED ISSUE #60: 🐛 [Bug] Translate Error Messages
## :bug: Bugreport

Translate all Error Messages in the Frontend.

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
Validation messages are translated to German now.

### Issues
- fixes #60

### Todo
Check if we also have to translate certain field-names (e.g. Password vs. Passwort)

### CODE REVIEW

File: frontend/src/vee-validate.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { configure, extend } from 'vee-validate'
+import { required, email, min } from "vee-validate/dist/rules"
+import i18n from './i18n'
+
+
+configure({
+  defaultMessage: (field, values) => {
+    console.log('defaultMessage', field, value)
```
Comment (ulfgebhardt) [inline frontend/src/vee-validate.js:8]:
```suggestion
```

Is this useful? Keep?

File: frontend/src/views/Layout/ContentFooter.vue [hunk 2]
```diff
@@ -23,30 +23,33 @@
...
-     <br>
-   <b-row >
-            <b-col class="nav-link text-center"  @click.prevent="setLocale('en')">{{ $t('languages.en') }}</b-col>
-            <b-col class="nav-link text-center" @click.prevent="setLocale('de')">{{ $t('languages.de') }}</b-col>
-        </b-row>
+    <br>
+    <br>
+    <b-row >
+      <b-col class="nav-link text-center"  @click.prevent="setLocale('en')">{{ $t('languages.en') }}XXXXX</b-col>
+      <b-col class="nav-link text-center" @click.prevent="setLocale('de')">{{ $t('languages.de') }}</b-col>
```
Comment (ulfgebhardt) [inline frontend/src/views/Layout/ContentFooter.vue:30]:
```suggestion
      <b-col class="nav-link text-center"  @click.prevent="setLocale('en')">{{ $t('languages.en') }}</b-col>
      <b-col class="nav-link text-center" @click.prevent="setLocale('de')">{{ $t('languages.de') }}</b-col>
```

What are the Xses for?

**Review by ulfgebhardt — CHANGES_REQUESTED** [review]

File: frontend/src/vee-validate.js (hunk 1 above)
Comment (Mogge) [reply frontend/src/vee-validate.js:8]: Thanks!

File: frontend/src/views/Layout/ContentFooter.vue (hunk 2 above)
Comment (Mogge) [reply frontend/src/views/Layout/ContentFooter.vue:30]: Ups.

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**